master/getting-started/kubernetes/installation/hosted/kubernetes-datastore/index.md

```diff
@@ -51,91 +50,135 @@ Ensure you have a cluster which meets the above requirements.  There may be addi
 
 ```
 kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/rbac-kdd.yaml
 ```
 
->[Click here to view the above yaml directly.](../rbac-kdd.yaml)
-
-### 1. {{site.prodname}} policy with {{site.prodname}} networking (Beta)
+>[Click here to view the above YAML directly.](../rbac-kdd.yaml)
 
-With Kubernetes as the {{site.prodname}} datastore, {{site.prodname}} has beta support for {{site.prodname}} networking.  This provides BGP-based
-networking with a full node-to-node mesh and/or explicit configuration of peers.
+### Option 1: {{site.prodname}} policy with {{site.prodname}} networking
 
-To install {{site.prodname}} with {{site.prodname}} networking, run one of the commands below based on your Kubernetes version.
-This will install {{site.prodname}} and will initially create a full node-to-node mesh.
+With Kubernetes as the {{site.prodname}} datastore, {{site.prodname}} supports 
+{{site.prodname}} networking.  This provides BGP-based networking with a full 
+node-to-node mesh and/or explicit configuration of peers.
 
-> **Note**: Calico v2.5.0 or later with Kubernetes backend requires Kubernetes
-> v1.7.0 or higher.
-{: .alert .alert-info}
-
-```
-kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/kubernetes-datastore/calico-networking/1.7/calico.yaml
-```
+To install {{site.prodname}} with BGP networking:
 
->[Click here to view the above yaml directly.](calico-networking/1.7/calico.yaml)
+1. Download [the {{site.prodname}} networking manifest](calico-networking/1.7/calico.yaml)
 
+2. If your Kubernetes cluster contains more than 50 nodes, or it is likely to grow to 
+   more than 50 nodes, edit the manifest to [enable Typha](#enabling-typha).
 
-#### {{site.prodname}} policy with {{site.prodname}} networking on kubeadm
+3. Make sure your cluster CIDR matches the `CALICO_IPV4POOL_CIDR` environment variable in the manifest.
+   The cluster CIDR is configured by the  `--cluster-cidr` option passed to the Kubernetes 
+   controller manager.  If you are using `kubeadm` that option is controlled by `kubeadm`'s
+   `--pod-network-cidr` option.
+   
+   > **Note**: {{site.prodname}} only uses the `CALICO_IPV4POOL_CIDR` variable if there is no
+   > IP pool already created.  Changing the variable after the first node has started has no 
+   > effect.
+   {: .alert .alert-info}
 
-The above manifests are compatible with kubeadm clusters initialized with a
-pod-network-cidr matching the default pool of `192.168.0.0/16`, as follows:
+4. Apply the manifest: `kubectl apply -f calico.yaml`
 
-```
-kubeadm init --pod-network-cidr=192.168.0.0/16
-```
+5. If your Kubernetes cluster has more than 100 nodes, we recommend disabling the 
+   node-to-node BGP mesh and configuring a pair of redundant route reflectors.
+   Due to limitations in the Kubernetes API, maintaining the node-to-node mesh 
```

Comment:
Doesn't Typha watch all nodes as well? Why does the RR use more CPU?

Reply:
Typha is optimised to squash inconsequential Node updates early in the pipeline, `confd` is more generic and does more work on each change.  We _do_ have an optimisation in place to stop `confd` polling nodes from every calico/node if the node-to-node mesh is disabled so that's why a RR makes such a  difference.

Reply:
Gotcha.

Seems like we should put some thought into confd as to whether or not we can implement similar logic to Typha for squashing duplicate updates.